### Question 1. Select the Tool

What's the name of the orchestrator you chose? **prefect**

### Question 2. Version

In [1]:
import prefect 
!prefect version

Version:             3.4.4
API version:         0.8.4
Python version:      3.12.2
Git commit:          0367d7aa
Built:               Thu, May 29, 2025 09:37 PM
OS/Arch:             linux/x86_64
Profile:             ephemeral
Server type:         ephemeral
Pydantic version:    2.11.5
Server:
  Database:          sqlite
  SQLite version:    3.46.0


What's the version of the orchestrator? **3.4.4**

### Question 3. Creating a pipeline

In [2]:
import os

In [8]:
os.system(f"python prefect_pipeline.py")

The number of records is 3403766.


0

How many records did we load? **3,403,766**

### Question 4. Data preparation